app/src/components/ModuleControls/index.js

```
@@ -1,49 +1,50 @@
 // @flow
 import * as React from 'react'
-import { connect } from 'react-redux'
-import ModuleData from './ModuleData'
-import TemperatureControls from './TemperatureControls'
+import TemperatureControl from './TemperatureControl'
 
-import { sendModuleCommand } from '../../robot-api'
-
-import type { State, Dispatch } from '../../types'
-import type { TempDeckModule, ModuleCommandRequest } from '../../robot-api'
+import type { TempDeckModule, ThermocyclerModule } from '../../robot-api'
 import type { Robot } from '../../discovery'
+import useSendModuleCommand from './useSendModuleCommand'
+import styles from './styles.css'
+import TemperatureData from './TemperatureData'
 
-type OP = {|
+type Props = {|
   robot: Robot,
```

This robot prop (previously) allowed the module temperature controls to be used even if the RPC API wasn't connected. With the addition of `useSendModuleCommand` that implicitly grabs the connected robot, the set temperature UI now fails silently if the user has navigated to the modules card without clicking the connect button first.

As discussed IRL, we should disable the set-temp button in the modules card if the current UI robot is not the connected robot (or there is no connected robot)